client/src/components/Tea/TeaEditor/TeaEditor.jsx

```diff
@@ -3,123 +3,125 @@ import React from "react";
 import { Link } from "react-router-dom";
 import DataList from "../../DataList";
 
-export const TeaEditor = props => {
-  return (
-    <div className="container">
-      {props.flash.name ? (
-        <div className="notification is-success">
-          {props.flash.name} has been succesfully saved.{" "}
-          <Link to={"../../tea/" + props.flash.teaID}>View details</Link>
-        </div>
-      ) : (
-        ""
-      )}
+export class TeaEditor extends React.Component {
+  render() {
+    return (
+      <div className="container">
+        {this.props.flash.name && (
+          <div className="notification is-success">
+            {this.props.flash.name} has been succesfully saved.{" "}
+            <Link to={"../../tea/" + this.props.flash.teaID}>View details</Link>
+          </div>
+        )}
 
-      <form onSubmit={e => props.handleFormSubmit(e)}>
-        <div className="field">
-          <label htmlFor="name">
-            Tea Name
-            <div className="control">
-              <input
-                className="input"
-                // className={
-                //   props.shouldMarkError("name") ? "input is-danger" : "input"
-                // }
-                type="text"
-                id="name"
-                onChange={props.handleNameChange}
-                value={props.name}
-                placeholder="Tea Name"
-                onBlur={props.handleBlur("name")}
-              />
-            </div>
-            {/* {props.shouldMarkError("name") ? (
+        <form onSubmit={this.props.handleFormSubmit}>
+          <div className="field">
+            <label htmlFor="name">
+              Tea Name
+              <div className="control">
+                <input
+                  className="input"
+                  // className={
```

Comment:
I know you have these comments in here for when you add validation, but as a general rule of thumb, you shouldn't check in commented-out code.